packages/stark-ui/src/modules/app-logout/app-logout.module.ts

```diff
@@ -1,14 +1,30 @@
 import { NgModule } from "@angular/core";
-import { StarkAppLogoutComponent } from "./components";
 import { MatIconModule } from "@angular/material/icon";
 import { MatTooltipModule } from "@angular/material/tooltip";
 import { MatButtonModule } from "@angular/material/button";
-import { TranslateModule } from "@ngx-translate/core";
+import { TranslateModule, TranslateService } from "@ngx-translate/core";
+import { StarkLocale } from "@nationalbankbelgium/stark-core";
+import { StarkAppLogoutComponent } from "./components";
 import { StarkSvgViewBoxModule } from "../svg-view-box/svg-view-box.module";
+import { translationsEn } from "./assets/translations/en";
+import { translationsFr } from "./assets/translations/fr";
+import { translationsNl } from "./assets/translations/nl";
+import { mergeUiTranslations } from "../../common/translations";
 
 @NgModule({
 	declarations: [StarkAppLogoutComponent],
 	exports: [StarkAppLogoutComponent],
 	imports: [MatIconModule, StarkSvgViewBoxModule, TranslateModule, MatTooltipModule, MatButtonModule]
 })
-export class StarkAppLogoutModule {}
+export class StarkAppLogoutModule {
+	/**
+	 * Class constructor
+	 * @param translateService - the translation service of the application
+	 */
+	public constructor(private translateService: TranslateService) {
```

Comment:
Why declaring variable translateService here ?
Couldn't you do 
```typescript
public constructor(translateService: TranslateService) {
  const english: StarkLocale = { languageCode: "en", translations: translationsEn };
  const french: StarkLocale = { languageCode: "fr", translations: translationsFr };
  const dutch: StarkLocale = { languageCode: "nl", translations: translationsNl };
  mergeUiTranslations(translateService, english, french, dutch);
}
```

This is the same for the other modules. Let me know what you think about this 😊

Reply:
Indeed, I'll fix it 👍